internal/services/datafactory/data_factory_linked_service_sftp_resource.go

```diff
@@ -103,31 +121,42 @@ func resourceDataFactoryLinkedServiceSFTP() *pluginsdk.Resource {
 				},
 			},
 
-			"skip_host_key_validation": {
-				Type:     pluginsdk.TypeBool,
-				Optional: true,
+			"passphrase": {
+				Type:          pluginsdk.TypeString,
+				Optional:      true,
+				Description:   "Specify the pass phrase or password to decrypt the private key if the key file or the key content is protected by a pass phrase",
+				ConflictsWith: []string{"password"},
```

Comment:
I think this should be marked as sensitive
```suggestion
			"passphrase": {
				Type:          pluginsdk.TypeString,
				Optional:      true,
				Sensitive:     true,
				Description:   "Specify the pass phrase or password to decrypt the private key if the key file or the key content is protected by a pass phrase",
				ConflictsWith: []string{"password"},
```

Reply:
Oh thanks I didn't notice the `Sensitive` prop.